pkg/tagger/remote/grpclog.go

```diff
@@ -0,0 +1,45 @@
+package remote
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"google.golang.org/grpc/grpclog"
+)
+
+type logLevel int
+
+const (
+	logLevelInfo logLevel = iota
+	logLevelWarn
+	logLevelErr
+)
+
+type redirectLogger struct {
+	level logLevel
+}
```

Comment:
it can be good to have a function `newRedirectLogger(level logLevel) *redirectLogger`